src/conn/DBConnection.h

```diff
@@ -30,10 +30,10 @@ class DBConnection
     
     private:
         //Connection Variables
-        sql::Connection *conn;
-        sql::PreparedStatement *prep_stmt;
+        sql::Connection *_conn;
+        sql::PreparedStatement *_prep_stmt;
```

Comment:
Hmmm in my comment for the protected variable in the Model class, I said use '_connection', but I think it is fine to leave these here.